Sources/BraintreePayPal/BTJSON+PayPal.swift

```diff
@@ -0,0 +1,36 @@
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
+extension BTJSON {
+
+    func asCreditFinancingAmount() -> BTPayPalCreditFinancingAmount? {
```

Comment:
```suggestion
    func asPayPalCreditFinancingAmount() -> BTPayPalCreditFinancingAmount? {
```
🧶 : May be overkill, but it could be a nice hint to calling files that it's a `PayPal` module extension method.

Reply:
Will do salute emoji